backend/library/views.py

```
@@ -13,13 +16,14 @@
 
 from django.http import HttpResponse
 
+import django_filters as df
 from core.helpers import get_sorted_requirement_nodes
 from core.models import StoredLibrary, LoadedLibrary
 from core.views import BaseModelViewSet
 from iam.models import RoleAssignment, Folder, Permission
 from library.validators import validate_file_extension
 from .helpers import update_translations, update_translations_in_object
```

Ruff (0.8.2)

`.helpers.update_translations_in_object` imported but unused

Remove unused import: `.helpers.update_translations_in_object`

(F401)